lib/models/project.js

```diff
@@ -338,7 +339,8 @@ Project.prototype.supportedInternalAddonPaths = function() {
     path.join(internalMiddlewarePath, 'tests-server'),
     path.join(internalMiddlewarePath, 'history-support'),
     path.join(internalMiddlewarePath, 'serve-files'),
-    path.join(internalMiddlewarePath, 'proxy-server')
+    path.join(internalMiddlewarePath, 'proxy-server'),
+    path.join(this.emberCliNodeModulesPath, 'ember-cli-legacy-blueprints')
```

Comment:
@stefanpenner there's actually no problem in adding it all the time because `ember-core` and `ember-data` will have priority for blueprint generation